### DISCUSSION TITLE
FR:  New Thermostat Card - allow a min/max temp for the different HVAC modes.

### DISCUSSION THREAD

**Discussion by TheRealBenForce** [home-assistant/frontend#19014] [Dashboard]
It would be really great to be able to set a max/min temp for the different HVAC modes and have the dial scale fit those or have some type of indicator where they are on the dial so you can "aim" for them. When using a touch input it is pretty easy to accidentally set the temperature to some insane number that no reasonable person would ever do.

For example, I never set my heat to anything less than 68F. This is what I set it to when I'm away from home. I typically keep it around 72F. Allow me to configure the card so 68F is the what the temp would be sent to if I slid the dial all the way to the lower value. On testing, it looks like 45F is how low it would currently go.

**Comment by frenck** [comment]
Hi there @TheRealBenForce 👋 

This is not a frontend feature request, as this requires quite a bit of backend work.
For that reason, please raise a feature request in our forums instead.

../Frenck
